docs/backend/guides/open-telemetry.md

```diff
@@ -0,0 +1,264 @@
+TELEMETRY_COLLECT_SECONDS | 30 | The interval in seconds between calls to collect metrics.
+
+When [running telemetry locally](../../tools/telemetry/running-telemetry-locally.md) with the open telemetry collector stack (defined in docker-compose.telemetry.yaml), the development configuration used is that of `config/telemetry/collector-config.dev.yaml`
+
+### Go package
+
+The main initialization code for registering telemetry can be found in `pkg/telemetry/init.go`
+
+The custom code for collecting database connection stats also lives within the package at `pkg/telemetry/db.go`.
+
+The code for hooking into the Go runtime library for memory stats lives within the package at `pkg/telemetry/runtime.go`
+
+### Instrumentation
+
+The ecosystem of Open Telemetry comes with many built in and 3rd party libraries for plugging into our existing code with little or no customization.
+
+#### Routing
+
+Milmove uses the Gorilla mux router so we can use the existing [otelmux library](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/github.com/gorilla/mux/otelmux) middleware for our non-swagger routes.
+
+```golang
+router.Use(otelmux.Middleware("auth"))
+```
+
+#### HTTP handler
+
+Similar to the otelmux library the [otelhttp library](https://github.com/trussworks/otelhttp) is designed to wrap HTTP handler functions. It has built in events (read & write) to report the request size, response size, and duration of the http request.  It is more configurable than the otelmux middleware allowing you to filter out certain routes and other controls.
+
+```golang
+otelhttp.NewHandler(router, "server-tls", []otelHTTP.Option{otelHTTP.ReadEvents, otelHTTP.WriteEvents})
+```
+
+Note: This implementation has been forked to the Trussworks GitHub to address bug fixes.
+
+#### SQL 
+
+The [XSAM/otelsql](https://github.com/XSAM/otelsql) library wraps the queries to our Postgres database and appends a span to the trace with the statement and duration information.  Query parameters may be hidden to not log sensitive information.  It can also be used to log calls to cursor rows when paging through larger result sets.
+
+#### Go
+
+The [Go runtime instrumentation library](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/runtime) primarily provides statistics on memory usage and garbage collection cycles that may degrade performance.
+
+## Data collection
+
+Data from the Milmove app is sent to the Open Telemetry collector, which processes the trace, metric, and log data and exports it to the services of our choice.  In AWS the collector exports to the Cloudwatch and AWS X-Ray services for storage and analysis.  When run locally the data is exported to the Elastic APM Server running Elasticsearch and Kibana. 
+
+### Traces
+
+*Common Trace fields*
+- ID
+- Parent ID
+- Trace ID
+- Name
+- Kind
+- Start time
+- End time
+- Status code
+- Status message
+
+#### HTTP Requests
+
+```
+http.client_ip
```

Comment:
I'm not sure where in the doc would be best since there's several places that mention them, but it might be good to indicate somewhere that most of these are semantic conventions defined by open telemetry: 

Trace: https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/
Metrics: https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/

Reply:
Great idea, their specs are very readable too.